packages/webapp/src/components/Filter/FilterButton/styles.module.scss

```diff
@@ -0,0 +1,50 @@
+.filterButtonContainer {
+  position: relative;
+  height: 100%;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+  background-color: var(--grey200);
+  border-radius: 4px;
+  border: 1px solid var(--grey400);
```

Comment:
Is this a compromise between Figma and the older app styles perhaps? The filter button is borderless in Figma but I see this border looks quite nice next to the old search input! Again just checking if it's intentional, and then I'll match with the collapsible search.

Reply:
Ohhh, I didn't see the borderless filter button! Apparently it's borderless in the desktop view, but it does have a border in mobile which is the screen I was looking at